contracts/Oracle.sol

```diff
@@ -195,7 +195,7 @@ contract Oracle is Ownable {
 
         bool iterate = true;
         uint256 roundBack = _roundsBack;
-        uint256 price;
+        uint256 price = 0;
```

Comment:
by default uint in solidity are initialized to 0, idk why crytic show error here 😂